optaplanner-website-root/content/download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -1,3 +1,6 @@
+:toc:
```

Comment:
From a `grep  -r 'toc-title' .` on all of `optaplanner-website`, there are no other TOC entries (i.e. The table of content will only have one entry, if I understand correctly).

Reply:
I do not get what are you talking about

Reply:
There are plenty of titles

Reply:
I see, so it derive titles and anchors from the `==` headers (ex: `== From 7.48+ to 8.0.0.Final`) and ignores `===` headers and below (ex: `=== OSGi metadata removed`). I thought it would use other `:toc:` entries which would be combined, hence why I look around the entire codebase for `:toc:` but couldn't find any (since we don't use table of contents elsewhere). LGTM then.

Reply:
Uh, I see. Yes that is a it strange I put the level of toc and it dos not consider = and only ==. Interesting thing I was not aware of that as well, but the good part that we do not to update it manually. Though I am a bit scary that it is not used in other parts of documentation.

Reply:
@Christopher-Chianelli can you please merge it?